app/utils/build-provider-asset-path.js

```diff
@@ -0,0 +1,16 @@
+import config from 'ember-get-config';
+import pathJoin from '../utils/path-join';
+
+export default function buildProviderAssetPath(providerId, assetName, isDomain) {
+    if (config.providerAssetsURL != 'local') {
+        return pathJoin(config.providerAssetsURL, providerId, assetName);
+    }
+    if (config.ASSET_SUFFIX) {
+        assetName = assetName.replace(/(\.[^\.]+)$/, function(match, $1) { return `-${config.ASSET_SUFFIX}${$1}`; });
+    }
+    const providerAssetPath = 'assets/osf-assets/files/preprints-assets';
```

Comment:
This should probably be in a configuration setting rather than in the middle of a file. Not that I expect it to change a lot, but just so it's centralized.